### DISCUSSION TITLE
Release/5.8.1

### DISCUSSION THREAD

**Discussion by Athanasius** [EDCD/EDMarketConnector#1825] [Announcement]
Release 5.8.1
===
This fixes a bug where the Cmdr/APIKey sections on Settings > EDSM would never be shown.

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.8.1'>Release/5.8.1</a>.</em>

**Comment by denisvvf** [comment]
Coriolis export is not working in this version.
